src/renderers/common/Renderer.js

```diff
@@ -1917,7 +1917,17 @@ class Renderer {
 			renderContext.depth = renderTarget.depthBuffer;
 			renderContext.stencil = renderTarget.stencilBuffer;
 			// #30329
-			renderContext.clearColorValue = this._clearColor;
+			renderContext.clearColorValue = this.backend.getClearColor();
```

Comment:
Color values assigned to `RenderContext.clearColorValue` must be in working color space and must honor premultiply alpha. Both were missing so far leading to incorrect clear colors when using `renderer.setClearColor()`.